lib/fuse_lowlevel.c

```diff
@@ -2305,9 +2305,9 @@ int fuse_lowlevel_notify_inval_inode(struct fuse_session *se, fuse_ino_t ino,
 	return send_notify_iov(se, FUSE_NOTIFY_INVAL_INODE, iov, 2);
 }
 
-int fuse_lowlevel_notify_expire_entry(struct fuse_session *se, fuse_ino_t parent,
-				      const char *name, size_t namelen,
-				      enum fuse_expire_flags flags)
+int fuse_lowlevel_notify_entry(struct fuse_session *se, fuse_ino_t parent,
```

Comment:
I believe this function should check that `flags` is fully understood (i.e., it should catch when we pass 42 to it instead of FUSE_LL_INVALIDATE).